Configuration.props

```diff
@@ -13,6 +13,10 @@
   />
   <PropertyGroup>
     <ProductVersion>9.1.199</ProductVersion>
+    <!-- TFV for all projects, try v4.7.2 but fallback if needed -->
+    <_TFV472>$([Microsoft.Build.Utilities.ToolLocationHelper]::GetPathToStandardLibraries('.NETFramework', 'v4.7.2', ''))</_TFV472>
```

Comment:
Do we want to change the name of this so that it is more generic? we might change to `v4.8` at sometime in the near future?